content/en/docs/kubernetes/operator/troubleshooting/automatic.md

```diff
@@ -0,0 +1,178 @@
+
+Check that the `opentelemetry-auto-instrumentation` `init-container` has started
+up correctly (or has even started up at all), by running the following command:
+
+```shell
+kubectl get events -n <your_app_namespace>
+```
+
+Which should result in output that looks like the following example:
+
+```text
+53s         Normal   Created             pod/py-otel-server-7f54bf4cbc-p8wmj    Created container opentelemetry-auto-instrumentation
+53s         Normal   Started             pod/py-otel-server-7f54bf4cbc-p8wmj    Started container opentelemetry-auto-instrumentation
+```
+
+If the output is missing `Created` or `Started` entries for
+`opentelemetry-auto-instrumentation`, there might be an issue with
+your auto-instrumentation configuration. This can be the result of any of the
+following:
+
+- The `Instrumentation` resource wasn’t installed or wasn’t installed
+  properly.
+- The `Instrumentation` resource was installed after the application was
+  deployed.
+- There’s an error in the auto-instrumentation annotation, or the annotation in
+  the wrong spot. See the next section.
+
+You might also want to check the output of the events command for any errors, as
+these might help point to your issue.
+
+### Check the auto-instrumentation configuration
+
+The auto-instrumentation annotation might have not been added
+correctly. Check for the following:
+
+- Are you auto-instrumenting for the right language? For example, did you
+  try to auto-instrument a Python application by adding a JavaScript
+  auto-instrumentation annotation instead?
+- Did you put the auto-instrumentation annotation in the right location? When
+  you’re defining a `Deployment` resource, there are two locations where you could
+  add annotations: `spec.metadata.annotations`, and
+  `spec.template.metadata.annotations`. The auto-instrumentation annotation
+  needs to be added to `spec.template.metadata.annotations`, otherwise it doesn't
+  work.
+
+### Check auto-instrumentation endpoint configuration
+
+The `spec.exporter.endpoint` configuration in the `Instrumentation` resource
+allows you to define the destination for your telemetry data. If you omit it, it
+defaults to `http://localhost:4317`, which causes the data to be dropped.
+
+If you’re sending out your telemetry to a [Collector](/docs/collector/),
+the value of `spec.exporter.endpoint` must reference the name of your
+Collector
+[`Service`](https://kubernetes.io/docs/concepts/services-networking/service/).
+
+For example: `http://otel-collector.opentelemetry.svc.cluster.local:4318`.
+
+Where:
+
+- `otel-collector` is the name of the OTel Collector Kubernetes
+  [`Service`](https://kubernetes.io/docs/concepts/services-networking/service/)
+- In addition, if the Collector is running in a different namespace, you must
+  append `opentelemetry.svc.cluster.local` to the Collector’s service name,
+  where `opentelemetry` is the namespace in which the Collector resides (it can
+  be any namespace of your choosing).
```

Comment:
```suggestion
Where `otel-collector` is the name of the OTel Collector Kubernetes [`Service`](https://kubernetes.io/docs/concepts/services-networking/service/).

In addition, if the Collector is running in a different namespace, you must append `opentelemetry.svc.cluster.local` to the Collector’s service name, where `opentelemetry` is the namespace in which the Collector resides. It can be any namespace of your choosing.
```